shared/README.md

```diff
@@ -12,8 +12,8 @@ App introduction goes here ...
 
 * Setup and boot the Docker containers:
 
-```
-./bin/envsetup
+```sh
+./bin/envsetup.sh
```

Comment:
The current agreed consensus is to omit the file extension for executable files. Any string reason to change that?

Reply:
1. I didn't know about the consensus. Is this documented?
2. Mainly for consistencies sake. all other scripts in the project have the `.sh` suffix.

Reply:
Let's go with the consistency here then let's add the extension `.sh`.

As for the consensus, it's not documented but it's just part of following what rails does as all built-in-Rail files in the bin folder do not have extensions:

![mia-web___Volumes_Workspace_mia-web__-_____app_assets_stylesheets__variables_scss__mia-web_](https://user-images.githubusercontent.com/696529/57207116-3cd87480-6ff5-11e9-845b-c55ef820f4f3.png)

But for our files, some had while some did not e.g. `envsetup` vs, `start.sh`

This can be fixed in another PR

Reply:
gotcha.